Build fail: "javax.xml.bind.annotation does not exist" (openjdk 11, maven 3.6)

## Observed behaviour

error: `package javax.xml.bind.annotation does not exist`

```
[INFO] asqatasun 4.1.0-SNAPSHOT ........................... SUCCESS [  2.147 s]
[INFO] engine 4.1.0-SNAPSHOT .............................. SUCCESS [  0.029 s]
[INFO] commons-entity 4.1.0-SNAPSHOT ...................... SUCCESS [  1.403 s]
[INFO] commons-entity-jpa 4.1.0-SNAPSHOT .................. SUCCESS [  0.473 s]
[INFO] asqatasun-api 4.1.0-SNAPSHOT ....................... FAILURE [  1.363 s]
(...)
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project asqatasun-api: Compilation failure: Compilation failure: 
[ERROR] engine/asqatasun-api/src/main/java/org/asqatasun/service/ProcessorService.java:[25,33] package javax.xml.bind.annotation does not exist
[ERROR] engine/asqatasun-api/src/main/java/org/asqatasun/service/ProcessorService.java:[34,2] cannot find symbol
[ERROR]   symbol: class XmlTransient
```


## Expected behaviour
```
[INFO] asqatasun-api 4.1.0-SNAPSHOT ....................... SUCCESS 
```

## Steps to reproduce the behaviour
```
cd ~
mv .m2 .m2_backup
git clone ...
cd Asqatasun
mvn clean install
```

##  Configuration

```
mvn --version
	Apache Maven 3.6.0
	Maven home: /usr/share/maven
	Java version: 11.0.3, vendor: Oracle Corporation, runtime: /usr/lib/jvm/java-11-openjdk-amd64
	OS name: "linux", version: "4.15.0-54-generic", arch: "amd64", family: "unix"

java -version
	openjdk version "11.0.3" 2019-04-16
```

--- a/engine/asqatasun-api/pom.xml
+++ b/engine/asqatasun-api/pom.xml
@@ -67,5 +67,11 @@
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
         </dependency>
+
+	<!-- Java Architecture for XML Binding (JAXB), Java 11+ support -->		
+	<dependency>
+		<groupId>javax.xml.bind</groupId>
+		<artifactId>jaxb-api</artifactId>
+	</dependency>
     </dependencies>
 </project>

--- a/pom.xml
+++ b/pom.xml
@@ -129,7 +129,7 @@
         <build-helper-maven-plugin.version>3.0.0</build-helper-maven-plugin.version>
         <buildnumber-maven-plugin.version>1.4</buildnumber-maven-plugin.version>
         <dependency-check-maven.version>3.1.2</dependency-check-maven.version>
-        <jacoco-maven-plugin.version>0.8.1</jacoco-maven-plugin.version>
+        <jacoco-maven-plugin.version>0.8.4</jacoco-maven-plugin.version>
         <markdown-page-generator-plugin.version>2.1.0</markdown-page-generator-plugin.version>
         <yuicompressor-maven-plugin.version>1.5.1</yuicompressor-maven-plugin.version>
         <jaxb-api.version>2.3.1</jaxb-api.version>

--- a/pom.xml
+++ b/pom.xml
@@ -132,6 +132,7 @@
         <jacoco-maven-plugin.version>0.8.1</jacoco-maven-plugin.version>
         <markdown-page-generator-plugin.version>2.1.0</markdown-page-generator-plugin.version>
         <yuicompressor-maven-plugin.version>1.5.1</yuicompressor-maven-plugin.version>
+        <jaxb-api.version>2.3.1</jaxb-api.version>
     </properties>
     <repositories>
         <repository>
@@ -1788,6 +1789,13 @@
                 <groupId>com.ruleoftech</groupId>
                 <artifactId>markdown-page-generator-plugin</artifactId>
                 <version>${markdown-page-generator-plugin.version}</version>
+            </dependency>
+
+            <!-- Java Architecture for XML Binding (JAXB), Java 11+ support -->		
+            <dependency>
+                <groupId>javax.xml.bind</groupId>
+                <artifactId>jaxb-api</artifactId>
+                <version>${jaxb-api.version}</version>
             </dependency>
         </dependencies>
     </dependencyManagement>

--- a/pom.xml
+++ b/pom.xml
@@ -1527,7 +1527,7 @@
             <dependency>
                 <groupId>ar.com.fdvs</groupId>
                 <artifactId>DynamicJasper</artifactId>
-                <version>4.0.3</version>
+                <version>4.0.4</version>
                 <exclusions>
                     <exclusion>
                         <groupId>jasperreports</groupId>

